### PR TITLE
home and not found styled and functional, updated read me

### PR DESCRIPTION
**Pull request by giovannifld**

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -25,81 +25,80 @@ const handleLogout = () => {
...
-          <Collapse isOpen={isOpen} navbar>
-            <Nav navbar className='navbar-nav'>
-              <NavItem className='nav-item'>
-                <NavLink to="/" className='custom-link'>Home</NavLink>
-              </NavItem>
-              { isLoggedIn && (
-                <>
+      <Navbar {...args}>
+        <div>
+        <NavbarBrand  href="/" className='navbar-brand'>
```
Comment (sjproctor) [inline src/components/Header.js]: Internal navigation (any links within React) should only ever come from react router with the `to` attribute.

File: src/components/Header.js [hunk 2]
```diff
@@ -25,81 +25,80 @@ const handleLogout = () => {
...
+            src={logo}
+          />
+        </NavbarBrand>
+        </div>
+        <div className='dropDownMenu'>
+        <NavbarToggler  onClick={toggle} className='navbar-toggler'/>
+        <Collapse isOpen={isOpen} navbar>
+          <Nav navbar className='navbar-nav'>
+            <NavItem className='nav-item'>
+              <NavLink to="/" className='custom-link'>Home</NavLink>
```
Comment (sjproctor) [inline src/components/Header.js:43]: Line 43 is a perfect example!

File: src/pages/Home.js [hunk 3]
```diff
@@ -1,8 +1,38 @@
 import React from 'react'
-
+import homePic from '../assets/HomePicture.png'
+import arrow from '../assets/arrow.png'
 function Home() {
-  return (
-    <div>Home</div>
+  return (
```
Comment (sjproctor) [inline src/pages/Home.js:6]: The component function declaration on line 4 should be refactored.

File: src/pages/NotFound.js [hunk 4]
```diff
@@ -1,8 +1,18 @@
 import React from 'react'
+import notFoundPic from '../assets/image-from-rawpixel-id-10092909-png.png'
 
 function NotFound() {
   return (
-    <div>NotFound</div>
+    <div className='not-found'>
```
Comment (sjproctor) [inline src/pages/NotFound.js:6]: Same with this component function declaration on line 4

**Review by sjproctor — COMMENTED** [review]
One issue with your nav and it looks like your component function declarations need to be refactored to `const ComponentName = () => {`

**Review by SunkissedQueen — APPROVED** [review]

**Pull request merged**